index.html

```diff
@@ -30,6 +30,7 @@
     <!--<link href="assets/stylesheets/bootstrap.min.css" rel="stylesheet">-->
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.100.2/css/materialize.min.css">
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Quicksand" rel="stylesheet">
```

Comment:
Let's move the logic for this to a separate component so each just has a single responsibility